tests/local_client_server_tcp_e2e/local_client_server_tcp_e2e.c

```diff
@@ -26,6 +26,23 @@ static TEST_MUTEX_HANDLE g_dllByDll;
 
 #define TEST_TIMEOUT 30 // seconds
 
+static int generate_port_number()
+{
+	int port_number;
+	const char* comp_time = __TIME__;
+	int comp_secs = 10 * (comp_time[6] - '0') + (comp_time[7] - '0');
+	int comp_mins = 10 * (comp_time[3] - '0') + (comp_time[4] - '0');
+	int comp_hours = 10 * (comp_time[0] - '0') + (comp_time[1] - '0');
```

Comment:
Actually this is the part that ensures that two instances of the tests don't fail if they start at the same time.
__TIME__ is assigned during compilation; since our exe, _valgrind and _helgrind builds don't happen in parallel, this is the most distinctive info we can use to avoid collisions